src/lib.rs

```diff
@@ -800,23 +801,27 @@ impl<T: Counter> Histogram<T> {
     ///
     /// println!("{:?}", hist.iter_percentiles(1).collect::<Vec<_>>());
     ///
-    /// assert_eq!(perc.next(), Some((hist.value_at_percentile(0.01), 0.01, 1, 1)));
+    /// assert_eq!(perc.next(), Some(IterationValue { value: hist.value_at_percentile(0.01),
+    ///     percentile: 0.01, count_at_value: 1, count_since_last_iteration: 1 }));
     /// // step size = 50
-    /// assert_eq!(perc.next(), Some((hist.value_at_percentile(50.0), 50.0, 1, 5000 - 1)));
+    /// assert_eq!(perc.next(), Some(IterationValue { value: hist.value_at_percentile(50.0),
+    ///     percentile: 50.0, count_at_value: 1, count_since_last_iteration: 5000 - 1 }));
     /// // step size = 25
-    /// assert_eq!(perc.next(), Some((hist.value_at_percentile(75.0), 75.0, 1, 2500)));
+    /// assert_eq!(perc.next(), Some(IterationValue { value: hist.value_at_percentile(75.0),
+    ///     percentile: 75.0, count_at_value: 1, count_since_last_iteration: 2500 }));
     /// // step size = 12.5
-    /// assert_eq!(perc.next(), Some((hist.value_at_percentile(87.5), 87.5, 1, 1250)));
+    /// assert_eq!(perc.next(), Some(IterationValue { value: hist.value_at_percentile(87.5),
+    ///     percentile: 87.5, count_at_value: 1, count_since_last_iteration: 1250 }));
     /// // step size = 6.25
-    /// assert_eq!(perc.next(), Some((hist.value_at_percentile(93.75), 93.75, 1, 625)));
+    /// assert_eq!(perc.next(), Some(IterationValue { value: hist.value_at_percentile(93.75),
+    ///     percentile: 93.75, count_at_value: 1, count_since_last_iteration: 625 }));
     /// // step size = 3.125
-    /// assert_eq!(perc.next(), Some((hist.value_at_percentile(96.88), 96.88, 1, 313)));
+    /// assert_eq!(perc.next(), Some(IterationValue { value: hist.value_at_percentile(96.88),
+    ///     percentile: 96.88, count_at_value: 1, count_since_last_iteration: 313 }));
     /// // etc...
     /// ```
-    pub fn iter_percentiles<'a>
-        (&'a self,
-         percentileTicksPerHalfDistance: isize)
-         -> iterators::HistogramIterator<'a, T, iterators::percentile::Iter<'a, T>> {
+    pub fn iter_percentiles<'a>(&'a self, percentileTicksPerHalfDistance: isize)
+            -> HistogramIterator<'a, T, iterators::percentile::Iter<'a, T>> {
```

Comment:
It was visually jarring so I just fixed it by hand (here and the few other function param re-wrappings). I added the `use` to shorten the line with `HistogramIterator` on it, but I can pull that out to a separate PR.